examples/keyed_list/src/random.rs

```diff
@@ -47,12 +47,12 @@ pub fn choose_two_distinct_mut<T>(items: &mut [T]) -> Option<(&mut T, &mut T)> {
     let (a, b) = items.split_at_mut(hi);
     Some((&mut a[lo], &mut b[0]))
 }
-
+#[allow(dead_code)]
```

Comment:
Please remove these functions entirely if they're no longer used. We don't want dead code in the example.